profiles/base-empty.yaml

```diff
@@ -62,3 +62,4 @@ empty_values:
   crafting_training_spells: {}
   restock_shop: {}
   theurgy_supply_levels: {}
+  caravan_training_skill: {}
```

Comment:
❓ Should this be `caravan_training_skills` (note the 's')?

Reply:
Doh